utils/stand_monitoring/poller/poller.py

```diff
@@ -1,88 +1,76 @@
 import json
-
 import polling
 import requests
-from requests import Response
 
 
 config_path = './config.json'
 with open(config_path, 'r') as f_config:
     config = json.load(f_config)
 
 
-def probe(urls: dict, payload: dict, response_patterns: dict):
+def probe(stands: dict, request_timeout: float, response_pattern: int):
     probe_result = {}
-    urls_post = urls['post']
-    urls_get = urls['get']
-
-    for payload_type, urls_list in urls_post.items():
-        for url in urls_list:
-            pattern = response_patterns[url] if url in response_patterns.keys() else response_patterns['default']
-            try:
-                response = requests.post(url, json=payload[payload_type])
-                probe_result[url] = assert_response(response, pattern)
-            except Exception:
-                probe_result[url] = False
-
-    for url in urls_get:
-        pattern = response_patterns[url] if url in response_patterns.keys() else response_patterns['default']
+
+    for url, model in stands.items():
+        stand_name = ' '.join([model, url])
         try:
-            response = requests.get(url)
-            probe_result[url] = assert_response(response, pattern)
+            response = requests.post(url+'/poller', json={}, timeout=request_timeout)
+            probe_result[stand_name] = response.status_code == response_pattern
         except Exception:
-            probe_result[url] = False
+            probe_result[stand_name] = False
 
     return probe_result
 
 
-def assert_response(response: Response, pattern: dict):
-    if 'code' in pattern.keys():
-        if pattern['code'] != response.status_code:
-            return False
-
-    return True
-
-
-def act(urls_status: dict, probe_result: dict):
-    bad_urls = [url for url, status in urls_status.items() if url in probe_result and status and not probe_result[url]]
-    if bad_urls:
-        notify(bad_urls)
+def act(stands_status: dict, probe_result: dict):
+    urls = {url: probe_result[url] for url, status in stands_status.items() if (url in probe_result and
+                                                                                status is not probe_result[url])}
+    if urls:
+        notify(urls)
 
 
-def notify(bad_urls: list):
+def notify(stands: dict, first_notification=False):
     channel = config['general']['notification']
     channel_config = config['notification'][channel]
 
-    bad_urls_str = '\n'.join(bad_urls)
-    notification = f'Following stand endpoints are unreachable:\n\n{bad_urls_str}'
+    good_stands = '\n'.join([stand for stand, status in stands.items() if status])
+    bad_stands = '\n'.join([stand for stand, status in stands.items() if not status])
+    notification = []
+    if first_notification:
+        notification.append('THIS IS FIRST NOTIFICATION AFTER LAUNCH.')
```

Comment:
Wording: `THIS IS FIRST NOTIFICATION AFTER LAUNCH` -> `DEMO STAND SERVICES HEALTH CHECK SCRIPT WAS LAUNCHED`